lib/policies/jwt/jwt.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
+const passport = require('passport');
+const JWTStrategy = require('passport-jwt').Strategy;
+const extractors = require('./extractors');
+const services = require('../../services');
+
+module.exports = function (params) {
+  const secret = params.secretFile ? fs.readFileSync(params.secretFile) : params.secret;
+  const extractor = extractors[params.jwtExtractor](params.jwtExtractorField);
+
+  passport.use(new JWTStrategy({
+    secretOrKey: secret,
+    jwtFromRequest: extractor,
+    audience: params.audience,
+    issuer: params.issuer
+  }, (jwtPayload, done) => {
+    if (!jwtPayload || !jwtPayload.sub) {
+      return done(null, false);
```

Comment:
I think we need an option here where we _don't_ validate the user in Express Gateway.  There are plenty of use cases where I just want to verify the integrity of the JWT token and pass it along.  If using an external Identity Provider (like Auth0), it will be a bit of a pain to have to import and synchronize those user records.